libcontainer/standard_init_linux.go

```diff
@@ -6,17 +6,16 @@ import (
 	"os"
 	"os/exec"
 
-	"github.com/opencontainers/runtime-spec/specs-go"
-	"github.com/opencontainers/selinux/go-selinux"
-	"github.com/sirupsen/logrus"
-	"golang.org/x/sys/unix"
-
 	"github.com/opencontainers/runc/libcontainer/apparmor"
 	"github.com/opencontainers/runc/libcontainer/configs"
 	"github.com/opencontainers/runc/libcontainer/keys"
 	"github.com/opencontainers/runc/libcontainer/seccomp"
 	"github.com/opencontainers/runc/libcontainer/system"
 	"github.com/opencontainers/runc/libcontainer/utils"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/selinux/go-selinux"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
```

Comment:
What is the reason for this change?

Usually in runc code we try to maintain imports grouped into three groups, in order:
- Go standard packages
- vendored packages
- local packages (i.e. github.com/opencontainers/runc)

I think it makes sense to not change this in the existing code.